File: angular-frontend/src/app/models/photo.ts
export class Photo {
    
    constructor(
        private _photoId: number,
        private _photoUrl: string,
        private _photoComment: string,
    ){}

    public get photoId(): number {
        return this._photoId;
    }

    public set photoId(value: number) {
        this._photoId=value;
    }

    public get photoUrl(): string {
        return this._photoUrl;
    }

    public set photoUrl(value: string) {
        this._photoUrl=value;
    }

    public get photoComment(): string {
        return this._photoComment;
    }

    public set photoComment(value: string) {
        this._photoComment=value;
    }

}

File: edge-service/src/main/java/com/ironhack/edgeservice/classes/Photo.java
package com.ironhack.edgeservice.classes;

public class Photo {

    /** ------------ Properties ------------ **/

    private Integer photoId;
    private String photoUrl;
    private String photoComment;

    /** ------------ Constructors ------------ **/

    public Photo() {
    }

    public Photo(String photoUrl, String photoComment) {
        setPhotoUrl(photoUrl);
        setPhotoComment(photoComment);
    }

    /** --------- Getters & Setters --------- **/

    public Integer getPhotoId() {
        return photoId;
    }

    public void setPhotoId(Integer photoId) {
        this.photoId = photoId;
    }

    public String getPhotoUrl() {
        return photoUrl;
    }

    public void setPhotoUrl(String photoUrl) {
        this.photoUrl = photoUrl;
    }

    public String getPhotoComment() {
        return photoComment;
    }

    public void setPhotoComment(String photoComment) {
        this.photoComment = photoComment;
    }
}
